Virtusize/Sources/Internal/API/VirtusizeAPIRequest.swift

```diff
@@ -177,25 +186,6 @@ extension APIRequest {
 		) else {
 			return nil
 		}
-/*
- Chirag: Belows Logic is for to check request Body
-        // Print JSON from data
-    do {
-     
-            if let json = try JSONSerialization.jsonObject(with: jsonData, options: []) as? [String: Any] {
-                // Convert JSON object back to Data for pretty printing
-                let prettyPrintedData = try JSONSerialization.data(withJSONObject: json, options: .prettyPrinted)
-                
-                // Convert the Data to a string
-                if let prettyPrintedString = String(data: prettyPrintedData, encoding: .utf8) {
-                    print(prettyPrintedString)
-                }
-            }
-        } catch {
-            print("Error converting JSON data to string: \(error.localizedDescription)")
-        }
-   */
-       
```

Comment:
I have no idea why this is commented out. I prefer not to have any unused and commented code in the SDK.

Reply:
Yes.This is good idea to remove. Thanks.